### PR TITLE
e2e error collector scenarios

### PR DESCRIPTION
**Pull request by muminmmd**
@sawantpritam please review

### CODE REVIEW

File: e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@Error_Collector
+Feature: Error Collector Plugin - Verify Design time scenarios
```
Comment (sawantpritam) [inline e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature]: No need of all this design time scenarios... this feature file can be deleted.

File: e2e-test/src/test/features/errorcollector/RunTimeScenarioErrorCollector.feature [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    And Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Expand Plugin group in the LHS plugins list: "Transform"
+    And Select plugin: "Wrangler" from the plugins list as: "Transform"
+    And Expand Plugin group in the LHS plugins list: "Error Handlers and Alerts"
+    And Select plugin: "Error Collector" from the plugins list as: "Error Handlers and Alerts"
+    And Align plugin: "ErrorCollector" on canvas moving by x offset: 80 and y offset: 200
+    And Expand Plugin group in the LHS plugins list: "Sink"
+    And Select plugin: "BigQuery" from the plugins list as: "Sink"
+    And Connect plugins: "BigQuery" and "Wrangler" to establish connection
+    #And Drag source as "BigQuery" having title "BigQuery" and sink4 as "ErrorCollector" having title "ErrorCollector" to establish connection
```
Comment (sawantpritam) [inline e2e-test/src/test/features/errorcollector/RunTimeScenarioErrorCollector.feature]: DELETE COMMENTED LINES. CHECK ALL FILES.

I SEE THIS IN YOUR EVERY PR. PLEASE PAY ATTENTION BEFORE RAISING PR.

File: e2e-test/src/test/java/io/cdap/plugin/errorcollector/actions/ErrorCollectorActions.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ * Error Collector related actions.
+ */
+
+public class ErrorCollectorActions {
+
+  static {
+    SeleniumHelper.getPropertiesLocators(ErrorCollectorLocators.class);
+  }
+
+  public static void dragAndDrop(String source, String sourceTitle, int index, String sink, String sinkTitle) {
```
Comment (sawantpritam) [inline e2e-test/src/test/java/io/cdap/plugin/errorcollector/actions/ErrorCollectorActions.java]: None of the below actions is needed. Delete file

File: e2e-test/src/test/java/io/cdap/plugin/errorcollector/locators/ErrorCollectorLocators.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Error Collector plugin Locators.
+ */
+
+public class ErrorCollectorLocators {
+
+  public static WebElement sinkNodeWithTitle(String sink, String sinkTitle) {
```
Comment (sawantpritam) [inline e2e-test/src/test/java/io/cdap/plugin/errorcollector/locators/ErrorCollectorLocators.java]: None of the below locator needed. Delete file

File: e2e-test/src/test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java [hunk 5]
```diff
@@ -0,0 +1,134 @@
+/*
```
Comment (sawantpritam) [inline e2e-test/src/test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: Remove TestSetupHooks file from PR.

File: e2e-test/src/test/resources/pluginParameters.properties [hunk 6]
```diff
@@ -0,0 +1,84 @@
+projectId=cdf-athena
```
Comment (sawantpritam) [inline e2e-test/src/test/resources/pluginParameters.properties]: Remove unused properties.

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature (hunk 1 above)
Comment (muminmmd) [reply e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature]: Feature file deleted.

File: e2e-test/src/test/features/errorcollector/RunTimeScenarioErrorCollector.feature (hunk 2 above)
Comment (muminmmd) [reply e2e-test/src/test/features/errorcollector/RunTimeScenarioErrorCollector.feature]: Comments removed.

File: e2e-test/src/test/java/io/cdap/plugin/errorcollector/actions/ErrorCollectorActions.java (hunk 3 above)
Comment (muminmmd) [reply e2e-test/src/test/java/io/cdap/plugin/errorcollector/actions/ErrorCollectorActions.java]: File deleted.

File: e2e-test/src/test/java/io/cdap/plugin/errorcollector/locators/ErrorCollectorLocators.java (hunk 4 above)
Comment (muminmmd) [reply e2e-test/src/test/java/io/cdap/plugin/errorcollector/locators/ErrorCollectorLocators.java]: File deleted

File: e2e-test/src/test/resources/pluginParameters.properties (hunk 6 above)
Comment (muminmmd) [reply e2e-test/src/test/resources/pluginParameters.properties]: Properties removed.

File: e2e-test/src/test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java (hunk 5 above)
Comment (muminmmd) [reply e2e-test/src/test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: Will remove it while squashing commits.

File: e2e-test/src/test/resources/errorMessage.properties [hunk 7]
```diff
@@ -0,0 +1,10 @@
+validationSuccessMessage=No errors found.
```
Comment (sawantpritam) [inline e2e-test/src/test/resources/errorMessage.properties]: Remove unused error messages.
I think only first 2 required.

File: e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  @Error_Collector-04
+  Scenario: Validate error collector with same testdata in all input fields
+    Given Open Datafusion Project to configure pipeline
+    And Expand Plugin group in the LHS plugins list: "Error Handlers and Alerts"
+    And Select plugin: "Error Collector" from the plugins list as: "Error Handlers and Alerts"
+    And Navigate to the properties page of plugin: "ErrorCollector"
+    And Replace input plugin property: "errorCollectorErrorMessageColumnName" with value: "ecErrorMessageColumnName"
+    And Replace input plugin property: "errorCollectorErrorCodeColumnName" with value: "ecErrorMessageColumnName"
+    And Replace input plugin property: "errorCollectorErrorEmitterNodeName" with value: "ecErrorMessageColumnName"
+    And Validate "Error Collector" plugin properties
```
Comment (sawantpritam) [inline e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature:43]: Add newline at the end of file.

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature (hunk 1 above)
Comment (sawantpritam) [reply e2e-test/src/test/features/errorcollector/DesignTimeErrorCollector.feature]: files have not been removed. Check all files.

https://github.com/sawantpritam/hydrator-plugins/pull/10/files

File: e2e-test/src/test/resources/errorMessage.properties (hunk 7 above)
Comment (muminmmd) [reply e2e-test/src/test/resources/errorMessage.properties]: > Remove unused error messages. I think only first 2 required.

Properties removed.

File: e2e-test/src/test/resources/pluginParameters.properties [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+dataset=test_automation
+e2e_source_table=e2e_error_collector
```
Comment (sawantpritam) [inline e2e-test/src/test/resources/pluginParameters.properties]: Don't use hardcoded pre-exiting table.
Use @BQ_SOURCE_TEST tag on scenario.
And use step
And Enter input plugin property: "table" with value: "bqSourceTable"

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/resources/pluginParameters.properties (hunk 9 above)
Comment (muminmmd) [reply e2e-test/src/test/resources/pluginParameters.properties]: Tags added

File: e2e-test/src/test/resources/pluginParameters.properties [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (sawantpritam) [inline e2e-test/src/test/resources/pluginParameters.properties]: Remove unused properties - e2e_source_table & csvFormat

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/resources/pluginParameters.properties (hunk 10 above)
Comment (muminmmd) [reply e2e-test/src/test/resources/pluginParameters.properties]: Property removed

**Pull request closed without merging**